### ISSUE TITLE
Highlight number, string and boolean placeholders differently

### ISSUE THREAD

**Issue by EmilyGraceSeville7cf** [tldr-pages/tldr-node-client#397]
## Expected behavior
Highlight integer/floating point number placeholders, string and boolean ones differently: add option to config to specify what color is required for such placeholder kind.

## Actual behavior
All placeholders are highlighted in one color.

## Log, debug output
Put any log or error output here.

## Environment
- Operating system - macOS/Linux/Windows (add version if needed)
- Node.js version (`node --version`)
- tldr-node-client version (`tldr --version`)
